app/views/shared/_navbar.html.erb

```diff
@@ -1,13 +1,12 @@
 <div class="navbar navbar-toggleable-md navbar-light">
   <div class="background"></div>
-  <div class="navbar-brand">
-    <%= render 'shared/logo_text' %>
-  </div>
   <button class="navbar-toggler navbar-toggler-right collapsed" type="button" data-toggle="collapse" data-target="#navbar-supported-content" aria-controls="navbar-supported-content" aria-expanded="false" aria-label="toggle-navigation">
     <span class="navbar-toggler-icon open"></span>
     <span class="close" aria-hidden="true">&times;</span>
   </button>
-
+  <div class="navbar-brand">
```

Comment:
I don't get how moving the hamburger to come before the rest of the navbar elements fixes it, but css is crazy sometimes lol 👍